Gemfile

```diff
@@ -48,16 +48,17 @@ group :development, :test do
 
   gem 'pry-rails'
   gem 'rb-readline'
+
+  gem 'rspec-rails', '~> 4.0.1'
```

Comment:
This goes in the dev and test groups as when you run "rake spec" you default load to the development environment. Rspec handles switching to the test environment when it starts up and loads the rest of the gems.